app/src/assets/localization/en/protocol_command_text.json

```
@@ -88,6 +97,7 @@
   "single_nozzle_layout": "single nozzle layout",
   "slot": "Slot {{slot_name}}",
   "stacker_display_name": "Stacker {{stacker_slot}}",
+  "store_labware_from_slot_to_stacker": "Store {{primaryDefinitionDisplayName}} from {{slotName}} to Flex Stacker",
```

more natural, matches `flex_stacker_store`
```suggestion
  "store_labware_from_slot_to_stacker": "Store {{primaryDefinitionDisplayName}} from {{slotName}} into Flex Stacker",
```